src/install/python/python-frameworks/python-agent-web2py.mdx

```diff
@@ -1,22 +1,12 @@
 ---
-title: Python agent and Web2py web framework
-tags:
-  - Agents
-  - Python agent
-  - Web frameworks and servers
-metaDescription: Details on using the Python agent with an application that uses the Web2py web framework.
-redirects:
-  - /docs/agents/python-agent/web-frameworks-servers/python-agent-web2py-web-framework
-  - /docs/python/python-agent-and-web2py
-  - /docs/agents/python-agent/web-frameworks/python-agent-and-web2py
-  - /docs/agents/python-agent/web-frameworks/python-agent-web2py
+componentType: appInfo
+optionType: framework
+headingText: Review start-up steps using Web2Py
 ---
 
-Web2py is an open source framework for creating web-based Python applications. You can integrate the Python agent with your Web2py app to monitor your app's performance. For general install procedures, see [Install the Python agent](/docs/agents/python-agent/installation-configuration/python-agent-installation).
+You need to integrate the Python agent with your application so that your app's major functions and web requests are received by the agent. To integrate the Python agent with your app, you can run the newrelic-admin script in front of your usual app startup command.
```

Comment:
```suggestion
You need to integrate the Python agent with your application so that your app's major functions and web requests are received by the agent. If using the runweb2py command, see instructions below, otherwise there's no specific steps you need to take to integrate with Web2Py so skip straight to the server specific instructions.
```